src/core/Akka/Serialization/Serialization.cs

```diff
@@ -350,9 +375,21 @@ public void AddSerializer(string name, Serializer serializer)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public void AddSerializationMap(Type type, Serializer serializer)
         {
+            if(_serializerMap.ContainsKey(type))
```

Comment:
This is the wrong place to log this, is it not? Isn't this our cache for which message type goes with which serializer?

Reply:
The configuration were parsed directly to these mapping dictionaries inside the constructor [here](https://github.com/akkadotnet/akka.net/blob/dev/src/core/Akka/Serialization/Serialization.cs#L185) and [here](https://github.com/akkadotnet/akka.net/blob/dev/src/core/Akka/Serialization/Serialization.cs#L192)